pkg/kf/apps/log_tailer.go

```diff
@@ -114,24 +114,36 @@ func (t *pushLogTailer) handleWatch() (bool, error) {
 	defer ws.Stop()
 
 	for e := range ws.ResultChan() {
-		app, ok := e.Object.(*v1alpha1.App)
-		if !ok {
+		switch e.Object.(type) {
```

Comment:
nit: for type-based switching I think it makes sense to use the variable assignment type:

```
switch object := e.Object.(type){
case *v1alpha1.App:
	// can use object instead of app because it's already cast
```